aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1380,15 +1344,19 @@ Tensor& nuclear_norm_out(Tensor& result, const Tensor& self, IntArrayRef dim, bo
   maybe_wrap_dims(dim_, self.dim());
 
   auto permutation = create_dim_backshift_permutation(dim_[0], dim_[1], self.dim());
-  auto permutation_reverse = create_reverse_permutation(permutation);
-
   Tensor p = self.permute(permutation);
-  at::sum_out(result, std::get<1>(at::svd(p, /*some=*/true, /*compute_uv=*/false)), -1, keepdim);
+  // Since we error out on svd_backward when we don't compute U and V, the backward pass for nuclear_norm
```

Comment:
I appreciate that this comment is just copy-pasted from the above, but can we take this opportunity to improve its language, too? Maybe something like

```
NOTE: U and V are computed only if gradmode is enabled, since the backward for nuclear norm uses svd_backward, which requires them. 
```

?